core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -426,6 +494,11 @@ private static String getDefaultHost() {
     }
   }
 
+  public Properties getOriginalProperties() {
```

Comment:
nit: this code base is mixed, but Kafka usually doesn't use `get` prefixes. Also, it might make sense to try to stay aligned with the changes that have been made to `AbstractConfig` in Kafka (there we only have `originals()` that returns a `Map` but `originalsProperties()` or `originalProperties()` would match).